wp_api/src/request.rs

```diff
@@ -278,6 +288,11 @@ impl WpNetworkHeaderMap {
             .and_then(|h| h.parse().ok())
     }
 
+    pub fn insert(&mut self, header_name: HeaderName, header_value: String) {
+        self.inner
+            .insert(header_name, header_value.parse().unwrap());
```

Comment:
Return an `Result`, instead of `unwrap`? Or, we can make this function only available to unit tests, which seems to be the only place it's currently used.

Reply:
Sure, I've moved it into the test mod. If we move it back out, it might make sense to have `header_value` be `std::ascii` to avoid the need for the `Result`?